rssalertbot/feed.py

```diff
@@ -144,8 +145,6 @@ async def _handle_fetch_failure(self, title, description):
             description (str): alert description
         """
 
-        self.log.error(f"{title}: {description}")
```

Comment:
I would definitely expect this method to still be logging errors, as the method description says.

Reply:
Ah, I can make that more clear.